mmv1/products/datastream/product.yaml

```diff
@@ -17,6 +17,9 @@ versions:
   - !ruby/object:Api::Product::Version
     name: ga
     base_url: https://datastream.googleapis.com/v1/
+  - !ruby/object:Api::Product::Version
+    name: beta
```

Comment:
New question about this block: I don't see a beta API for datastream: https://cloud.google.com/datastream/docs/reference/rest

Are these fields actually beta-only? I also see (most of) them in the GA docs; maybe we could just add them directly as GA fields? (Which would maybe render this block unnecessary as a side effect?)

Reply:
To my knowledge they are. I had a recent conversation with some folks on the datastream team and it was mentioned that SQL Server is not official released as GA. Also, if you take a look at the console, SQL Server is listed as `PREVIEW` when creating a stream. For those reasons, I elected to make the PR beta only, but I'm open to adding it to the `google` provider instead of `google-beta` is that is preferred.

![image](https://github.com/GoogleCloudPlatform/magic-modules/assets/34457149/eb0b0ab8-b707-4c0c-99a8-3fc4fd46f1dc)

Reply:
I've removed the `min_version: beta` and the the beta provider in the product.yaml in this commit: 939b4f1e8962c3ac939f24d2fc98170f48bdc4d7

Let me know if you want to revert it based on the above.

Reply:
np... reverted